examples/graph2d/21_barsWithEnd.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <title>Graph2d | Bar Graph Example</title>
+
+    <style type="text/css">
+        body, html {
+            font-family: sans-serif;
+        }
+    </style>
+
+    <script src="../../dist/vis.js"></script>
+    <link href="../../dist/vis-timeline-graph2d.min.css" rel="stylesheet" type="text/css" />
+</head>
+<body>
+<h2>Graph2d | Bar Graph With End Example</h2>
+<div style="width:700px; font-size:14px; text-align: justify;">
+    This example shows how you can plot a bar chart and supply an X2 value to have it fill.
+</div>
+<br />
+
+<div id="visualization"></div>
+
+<script type="text/javascript">
+
+    var container = document.getElementById('visualization');
+    var items = [
+        {x: '2014-06-11 00:00:00', x2: '2014-06-13 00:00:00', y: 10 },
+        {x: '2014-06-13 00:00:00', x2: '2014-06-14 00:00:00', y: 15 },
+        {x: '2014-06-15 00:00:00', x2: '2014-06-16 00:00:00', y: 14 },
+        {x: '2014-06-16 00:00:00', x2: '2014-06-17 00:00:00', y: 17 },
+        {x: '2014-06-17 00:00:00', x2: '2014-06-18 00:00:00', y: 20 },
+        {x: '2014-06-18 00:00:00', x2: '2014-06-25 00:00:00', y: 25 },
+    ];
+
+    var dataset = new vis.DataSet(items);
+    var options = {
+        style:'bar',
```

Comment:
What happens with other chart types like line charts? I suspect to `x2` to be ignored!?

Yan you please also update this example to use the new parameter: /examples/graph2d/11_barsSideBySideGroups.html